src/main/java/com/openlattice/chronicle/constants/AppComponent.java

```diff
@@ -18,4 +18,13 @@
     public String toString() {
         return component;
     }
+
+    public static AppComponent fromString(String name) {
```

Comment:
We dug our own whole here, would have be better to just use default serialization, but I am guessing these are used elsewhere?

Reply:
Correct. 

I added this in order to resolve an api request string parameter to one of the enums.